listenbrainz/webserver/views/user_timeline_event_api.py

```diff
@@ -154,11 +154,14 @@ def create_user_notification_event(user_name):
     metadata = NotificationMetadata(creator=creator['musicbrainz_id'], message=message)
 
     try:
-        db_user_timeline_event.create_user_notification_event(user['id'], metadata)
+        event = db_user_timeline_event.create_user_notification_event(user['id'], metadata)
     except DatabaseException:
         raise APIInternalServerError("Something went wrong, please try again.")
 
-    return jsonify({'status': 'ok'})
+    event_data = event.dict()
+    event_data['created'] = event_data['created'].timestamp()
```

Comment:
do we use a timestamp elsewhere in this API? (rather than a rfc 3339 date)

Reply:
The retrieve endpoint accepts timestamps:

https://github.com/metabrainz/listenbrainz-server/blob/331c9a93dd60cfa5174c382699ac69e729d5caa9/listenbrainz/webserver/views/user_timeline_event_api.py#L238-L239

The other create event endpoints also return a timestamp, for one example:

https://github.com/metabrainz/listenbrainz-server/blob/331c9a93dd60cfa5174c382699ac69e729d5caa9/listenbrainz/webserver/views/user_timeline_event_api.py#L221-L226